_config.yml

```diff
@@ -90,6 +90,7 @@ sidebar_categories:
   Shipping:
     - docs/advanced/shipping/overview
     - docs/advanced/shipping/custom-shipping-information
+    - docs/advanced/shipping/display-a-map
```

Comment:
I'd suggest that we move it under the more generic `Features` section, because developers may need it for a store locator or another use case not related to Shipping.

Reply:
I've done it in b740901